ores/ores.py

```diff
@@ -6,6 +6,11 @@
 * celery_worker -- Starts a "ScoreProcessor" celery worker
 * precached -- Starts a daemon that requests scores for revisions as they happen
 
+You can also launch a set of production like applications
+
+* applications.wsgi -- A wsgi server
+* applications.celery -- A celery worker
+
```

Comment:
Why not have different entry points be specified in the setup.py, instead of maintaining this separate file?